pegasus/sites.v3/code.org/public/global/fa/videos.haml

```diff
@@ -0,0 +1,49 @@
+---
+theme: responsive_full_width
+video_player: true
+set_dir: true
+---
+
+:ruby
+  @header["title"] = hoc_s(:curriculum_name_csf)
+
+- video_page_index = 3
+- video_page_section_title = [hoc_s(:video_library_page_title_ai_works), hoc_s(:video_library_page_title_computers_work), hoc_s(:video_library_page_title_internet_works)]
+- video_page_section_desc = [hoc_s(:video_library_page_description_ai_works), hoc_s(:video_library_page_description_computers_work), hoc_s(:video_library_page_description_internet_works)]
+
+%link{href: "/css/generated/design-system-pegasus.css", rel: "stylesheet"}
+%link{href: "/css/generated/page/video-page-styles.css", rel: "stylesheet"}
+
+%section.hero-banner-basic.overlay
+  .wrapper.flex-container.justify-space-between.align-items-center.wrap
+    .text-wrapper
+      %h1.white
+        =hoc_s(:video_library_page_main_title)
+      %p.white
+        =hoc_s(:video_library_page_hero_description)
+    %figure
+      %img{src: "/images/banners/video-logo.png", alt: ""}
+    .clear
+
+- video_page_index.times do |index|
+  %section.centered
+    .wrapper
+      %h2
+        = video_page_section_title[index]
+      %p
+        = video_page_section_desc[index]
+      - if index === 0
+        = view :carousel_how_ai_works_video
+      - if index === 1
+        = view :carousel_computer_works_video
+      - if index === 2
+        = view :carousel_internet_video
+  = view :section_divider_line
+
+%section.centered
+  %p
+    =hoc_s(:video_library_page_footer_description)
+  %a.link-button.secondary{href: "https://www.youtube.com/channel/UCJyEBMU1xVP2be1-AoGS1BA", target: "_blank", rel: "noopener noreferrer"}
+    =hoc_s(:video_library_page_footer_cta)
+
+= view :swiper_page_videos
```

Comment:
It looks like the swiper_page_videos JS file [hard codes the carousels](https://github.com/code-dot-org/code-dot-org/blob/cb4b87c9e92bbdf792ecf526b617aeebfc50d772/apps/src/sites/code.org/pages/views/swiper_page_videos.js#L11-L19). As not all of the carousels are on this page, it's crashing. You can probably just filter out the non-existant elements in that JS file to fix